scss/_patterns_tab-buttons.scss

```diff
@@ -0,0 +1,37 @@
+@mixin vf-p-tab-buttons {
+  .p-tab-buttons {
+    &.is-dense .p-tab-buttons__button {
```

Comment:
Doesn't linter warn about this line here? It seems like something more specific that `.p-tab-buttons__button`, so probably could be defined after it. Also to make sure the styles are overridden?

Reply:
@bartaz hmmm - it _did_ warn about this being after buttons in a previous version of this, so we moved it here. It doesn't warn currently, but I've moved it after the button, and it no longer warns, so, everyone wins!